README.md

```diff
@@ -1,5 +1,21 @@
-# tispark
-Running Spark SQL on top of TiDB / TiKV.
+## What is TiSpark?
+
+TiSpark is a thin layer built for running Apache Spark on top of TiDB/TiKV to answer the complex OLAP queries. It takes advantages of both the Spark platform and the distributed TiKV cluster, at the same time, seamlessly glues to TiDB, the distributed OLTP database, to provide a Hybrid Transactional/Analytical Processing (HTAP) to serve as a one-stop solution for online transactions and analysis.
+
+## TiSpark Architecture
+
+![architecture](./docs/architecture.png)
+
+
+- TiSpark integrates with Spark Catalyst Engine deeply. It provides precise control of the computing, which allows Spark read data from TiKV efficiently. It also supports index seek, which improves the performance of the point query execution significantly.
+
+- It utilizes several strategies to push down the computing to reduce the size of dataset handling by Spark SQL, which accelerates the query execution. It also uses the TiDB built-in statistical information for  the query plan optimization.
```

Comment:
please remove the extra space b/w "for" and "the"
"information for[here] the query plan optimization"